rust/src/transformations/polars/make_clamp/mod.rs

```diff
@@ -0,0 +1,120 @@
+use num::One;
+use polars::prelude::*;
+use std::rc::Rc;
+
+use crate::core::{Function, MetricSpace, StabilityMap, Transformation};
+use crate::domains::{Bounds, Context, DataTypeFrom, DynSeriesAtomDomain, ExprDomain, ExprMetric};
+use crate::error::*;
+use crate::traits::{CheckAtom, DistanceConstant, TotalOrd};
+
+/// Make a Transformation that returns a clip(<bounds>) expression for a LazyFrame.
+///
+/// # Arguments
+/// * `input_domain` - Expr domain
+/// * `input_metric` - The metric space under which neighboring LazyFrames are compared
+/// * `bounds` - bounds to be applied in clamp operation.
+pub fn make_clamp_expr<
+    M,
+    C: Context,
+    TA: 'static + Clone + TotalOrd + CheckAtom + NumericNative + DataTypeFrom,
+>(
+    input_domain: ExprDomain<C>,
+    input_metric: M,
+    bounds: (TA, TA),
+) -> Fallible<Transformation<ExprDomain<C>, ExprDomain<C>, M, M>>
+where
+    M: ExprMetric<C>,
+    M::Distance: DistanceConstant<M::Distance> + One + Clone,
+    (ExprDomain<C>, M): MetricSpace,
+    i32: From<TA>,
+{
+    let mut output_domain = input_domain.clone();
+
+    let active_column = input_domain.active_column()?;
+    // retrieve active series
+    let active_series = output_domain
+        .lazy_frame_domain
+        .try_column_mut(active_column.clone())?;
+
+    // add bounds to atom domain
+    let mut atom_domain = active_series.atom_domain()?.clone();
+    atom_domain.bounds = Some(Bounds::new_closed(bounds)?);
+
+    // update element domain in active series
+    active_series.element_domain = Rc::new(atom_domain) as Rc<dyn DynSeriesAtomDomain>;
+
+    let margins = output_domain
```

Comment:
Add a comment here explaining that margins could be preserved, but this functionality hasn't been implemented.